Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -233,22 +242,47 @@ open class AVFoundationPlayback: Playback {
     fileprivate func setupPlayer() {
         if let asset = self.asset {
             let item: AVPlayerItem = AVPlayerItem(asset: asset)
+            
+            #if os(iOS)
             player = AVPlayer(playerItem: item)
```

Comment:
Guys, I don't think that we need this `kLoop` anymore.
I suggest use the same structure for tvOS and iOS, `AVQueuePlayer` or `AVPlayer`.
Let's go with `AVPlayer` for both? What do you think?

Reply:
Removed.